benchmark/CMakeLists.txt

```diff
@@ -211,6 +211,9 @@ set(SWIFT_BENCHMARK_NUM_ONONE_ITERATIONS "" CACHE STRING
 # Options for the default (= empty) configuration
 set(BENCHOPTS "-whole-module-optimization")
 
+set(SWIFT_BENCHMARK_GENERATE_OPT_VIEW FALSE CACHE BOOL
```

Comment:
This should be an option.

Reply:
Also, if there is a requirement on pygments and pyyaml packages being around, if this option is set, there should be a configure check (not here, I would add it into a macro in ./benchmarks/cmake).

Reply:
I am thinking now that the check for the python modules should go under LLVM and then communicated to swift cmake through the llvm-config mechanism you showed me yesterday.  Something like LLVM_OPT_VIEWER_MODULES_AVAILABLE or something like that.

Reply:
SGTM